src/main/java/com/synopsys/integration/alert/channel/jira/common/JiraGlobalTestAction.java

```diff
@@ -45,8 +49,14 @@ public MessageResult testConfig(String configId, FieldModel fieldModel, FieldAcc
                 throw new AlertException("User did not match any known users.");
             }
 
-            if (isAppCheckEnabled(registeredFieldValues) && isAppMissing(registeredFieldValues)) {
-                throw new AlertException(String.format("Please configure the %s plugin for your server.", getChannelDisplayName()));
+            if (isAppCheckEnabled(registeredFieldValues)) {
+                if (isUserAdmin(registeredFieldValues)) {
+                    throw new AlertException("The configured user must be an admin if 'Plugin Check' is enabled");
+                }
+
+                if (isAppMissing(registeredFieldValues)) {
+                    throw new AlertException(String.format("Please configure the %s plugin for your server.", getChannelDisplayName()));
```

Comment:
I think this exception is misleading, I don't think using the channel display name is clear enough. 

We probably want something like this `Please configure the 'Alert Issue Property Indexer' app for you server.` That way they know exactly the name of the app to look for in the marketplace.

Reply:
Good idea. This text predated the current PR. It was just moved to its own if.